modules/op-release-notes-1-14.adoc

```diff
@@ -0,0 +1,290 @@
+
+* With this update, {tekton-chains} uses the `v1` version of pipeline run and task run formats internally.
+
+[id="tekton-results-new-features-1-14_{context}"]
+=== {tekton-results}
+
+* With this update, if {tekton-results} is installed, {tekton-results} records the summary and record data for pipeline runs started using {pac}.
+
+* With this update, {tekton-results} provides up to 100 megabytes of logging information for a pipeline or task.
+
+* With this update, any authenticated user can view the `tekton-results-api-service` route in the openshift-pipelines namespace to interact with Tekton Results using a REST API.
+
+* With this update, the {tekton-results} API includes a new endpoint for fetching summary and aggregation for a list of records.
+
+* With this update, the `GetLog` endpoint of the {tekton-results} API returns raw bytes with the `text/plain` content type.
+
+* With this update, you can optionally specify a custom CA certificate in the `options.configMaps.tekton-results-api-config.data.config.DB_SSLROOTCERT` spec in the `TektonResult` CR. In this case, Tekton Results requires an SSL connection to the database server and uses this certificate for the connection. However, if you use this spec, you must also use alternate configuration parameters for other {tekton-results} settings instead of the parameters described in the {tekton-results} documentation, as listed in the following table. Both the regular and the alternate parameter specs are within the `TektonResult` CR.
++
+.Alternate configuration parameters for {tekton-results}
+[options="header"]
+|===
+
+| Regular parameter spec | Alternate parameter spec
+
+| `logs_api` | `options.configMaps.tekton-results-api-config.data.config.LOGS_API`
+| `log_level` | `options.configMaps.tekton-results-api-config.data.config.LOG_LEVEL`
+| `db_port` | `options.configMaps.tekton-results-api-config.data.config.DB_PORT`
+| `db_host` | `options.configMaps.tekton-results-api-config.data.config.DB_HOST`
+| `logs_path` | `options.configMaps.tekton-results-api-config.data.config.LOGS_PATH`
+| `logs_type` | `options.configMaps.tekton-results-api-config.data.config.LOGS_TYPE`
+| `logs_buffer_size` | `options.configMaps.tekton-results-api-config.data.config.LOGS_BUFFER_SIZE`
+| `auth_disable` | `options.configMaps.tekton-results-api-config.data.config.AUTH_DISABLE`
+| `db_enable_auto_migration` | `options.configMaps.tekton-results-api-config.data.config.DB_ENABLE_AUTO_MIGRATION`
+| `server_port` | `options.configMaps.tekton-results-api-config.data.config.SERVER_PORT`
+| `prometheus_port` | `options.configMaps.tekton-results-api-config.data.config.PROMETHEUS_PORT`
+| `gcs_bucket_name` | `options.configMaps.tekton-results-api-config.data.config.GCS_BUCKET_NAME`
+
+|===
++
+For the settings not listed in this table, use the regular parameter specs in the documentation,
```

Comment:
the table covers specs, why are we calling them settings in this sentence?

Are we talking abt regular parameter specs mentioned above or those spec are mentioned somehwre else in the doc? If they are mentioned somehwere else, may be we can provide a cross-reference??